.circleci/config.yml

```diff
@@ -115,34 +115,14 @@ jobs:
           name: Restore PNPM Package Cache
           keys:
             - pnpm-packages-v2-{{ checksum "pnpm-lock.yaml" }}
-      # TODO remove me after ci builder updated
-      - run:
-          name: Install pnpm package manager
-          command: |
-            npm i pnpm --global
-      # TODO remove me after ci builder updated
-      # A github dep clones-with-immutable-args is installed via github
-      # packages installed via npm via github automatically run postpack scripts
-      # their postpack script happens to use yarn so we need it here
-      - run:
-          name: Install yarn package manager
-          command: |
-            npm i yarn@1 --global
       - run:
           name: Install dependencies
-          command: pnpm install --frozen-lockfile
+          command: pnpm install --frozen-lockfile --prefer-offline
       - save_cache:
           name: Save PNPM Package Cache
           key: pnpm-packages-v2-{{ checksum "pnpm-lock.yaml" }}
           paths:
             - "node_modules"
-            - "packages/chain-mon/node_modules"
-            - "packages/common-ts/node_modules"
-            - "packages/contracts-bedrock/node_modules"
-            - "packages/core-utils/node_modules"
-            - "packages/replica-healthcheck/node_modules"
-            - "packages/sdk/node_modules"
-            - "packages/contracts-ts/node_modules"
```

Comment:
these can go away and then anything that needs node modules can just `pnpm i --offline-only` to instantly install from cache

Reply:
will move to it's own pr though

Reply:
This own pr is now this pr

Reply:
Doing --prefer-offline instead of --offline-only just because it makes it less fragile to breaking with the same upside